doc/source/user_guide/ug_case_settings.rst

```diff
@@ -513,37 +517,37 @@ zbgc_nml
    "``ratio_S2N_sp``", "real", "algal S to N in mol/mol small plankton", "0.03"
    "``restore_bgc``", "logical", "DEPRECATED", "``.false.``"
    "``R_dFe2dust``", "real", "g/g :cite:`Tagliabue09`", "0.035"
-   "``scale_bgc``", "logical", "", "``.false.``"
+   "``scale_bgc``", "logical", "initialize bgc by scaling with salinity", "``.false.``"
    "``silicatetype``", "real", "mobility type between stationary and mobile silicate", "-1.0"
-   "``skl_bgc``", "logical", "DEPRECATED", "``.false.``"
-   "``solve_zbgc``", "logical", "", "``.false.``"
+   "``skl_bgc``", "logical", "DEPRECATED: skeletal layer biogeochemistry", "``.false.``"
+   "``solve_zbgc``", "logical", "solve z-biogeochemistry reactions", "``.false.``"
    "``solve_zsal``", "logical", "DEPRECATED", "``.false.``"
-   "``tau_max``", "real", "long time mobile to stationary exchanges", "7776000."
-   "``tau_min``", "real", "rapid module to stationary exchanges", "3600."
+   "``tau_max``", "real", "long time mobile to stationary exchanges (s)", "604800.0"
```

Comment:
The Icepack default is 7776000 and that's what we run with.  Is that value correct?

Reply:
I've changed it to be consistent with MPAS-Si.

Reply:
We're not documenting MPAS-Si.  What should the default be in Icepack?  Should we be using 604800 in Icepack and CICE then?  We'll need to update the code too if these defaults should change.

Reply:
Happy to make the change, just want to make sure.